bucket/download.py

```diff
@@ -94,13 +94,17 @@ def askForBucket():
 
 # Download training images.
 image_files = [image for image in annotations.keys()]
+image_files_exceptions = []
 for file in image_files:
     filename = os.path.join(output_dir, file)
     print('saving: {}'.format(file))
     print('to: {}'.format(filename))
     try:
         cos.Object(credentials_1['bucket'], file).download_file(filename)
     except Exception, err:
-        print("Filename: {0}, Client Error: {1}".format(filename, err))
+        image_files_exceptions.append(file)
+        print("File: {0}, Client Error: {1}".format(file, err))
 
-prepare_data.main(output_dir, output_dir)
\ No newline at end of file
+print("there're {0} image(s) that couldn't be downloaded".format(len(image_files_exceptions)))
```

Comment:
The `image_files_exceptions` array is a bit extra if we aren't going to use it in `prepare_data`